pds_doi_service/core/actions/test/draft_test.py

```diff
@@ -10,7 +10,8 @@
 from pds_doi_service.core.actions.release import DOICoreActionRelease
 from pds_doi_service.core.entities.doi import DoiStatus, ProductType
 from pds_doi_service.core.input.exceptions import InputFormatException, WarningDOIException
-from pds_doi_service.core.outputs.osti import DOIOstiRecord, DOIOstiXmlWebParser, DOIOstiJsonWebParser
+from pds_doi_service.core.outputs.osti.osti_record import DOIOstiRecord
+from pds_doi_service.core.outputs.osti.osti_web_parser import DOIOstiXmlWebParser, DOIOstiJsonWebParser
```

Comment:
Are these imports used in `draft_test.py`?

Reply:
Yup,

`DOIOstiRecord` -> line 388,
`DOIOstiXmlWebParser` -> line 45 (and many others)
`DOIOstiJsonWebParser` -> line 339 (among others)

Reply:
Thanks for addressing my total and abject laziness 🥱